style_rwd.css

```diff
@@ -2,21 +2,233 @@
     img.main_photo {
         display: none;
     }
-    .catch_phrase{
-        max-width: 767px;
+    h2 {
+        margin-block-start: 0;
+        margin-block-end: 0;
+    }
+    .nav_margin {
+        margin: 0 ;
+    }
+    .margin_header{
+        margin: 0;
+    }
+    .margin_menu {
+        margin: 0;
+    }
```

Comment:
better:
```css
.nav_margin, .margin_header, .margin_menu {
margin: 0;
}
```
almost best:
```css
.m-0 {
margin: 0;
}
```

Reply:
switched for the first one :) included also h2 part